containers/datalab/content/GCSbackup.sh

```diff
@@ -123,7 +123,11 @@ fi
 # test and create bucket if necessary
 gsutil ls gs://${gcs_bucket} &>/dev/null || {
   echo "Bucket '${gcs_bucket}' was not found. Creating it.."
-  gsutil mb gs://"${gcs_bucket}"
+  gsutil mb gs://"${gcs_bucket}" || {
```

Comment:
In addition to not trying to create the "...appspot.com" bucket (which we know would always fail), the other difference between what I proposed and the current approach is that we would not try to create the project bucket if the user has explicitly specified a gcs_bucket, regardless of whether or not that gcs_bucket exists.